src/styles/main.scss

```
@@ -11,4 +11,37 @@ body {
 }
 
 @import "_header.scss";
-@import "_catalog.scss";
+// @import "_catalog.scss";
```

Не совсем пойму, зачем ты решил писать все стили в main, если нам рекомендуют создавать отдельно scss файлы для каждого БЭМ-блока и подключать их в main, т.е. кроме `@import`  в main ничего не должно быть)
```suggestion
// @import "_catalog.scss";
```